### PR TITLE
Update live-response.md

### LINKED ISSUE #13206: is the 'scan' command a quick scan, or a full scan, or a quick scan then a full scan?
In...

https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/live-response?view=o365-worldwide#advanced-commands
...the current description is too vague;

> Runs an antivirus scan to help identify and remediate malware.

I have used the 'scan' command, and it completed a quick scan.

But I don't know if it will perform a full scan.

The document must remove this ambiguity.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 81b2a4c2-e698-dda0-9919-80ca2d05b358
* Version Independent ID: 19b5f609-fcaa-cb5b-96b7-462285fd32e7
* Content: [Investigate entities on devices using live response in Microsoft Defender for Endpoint](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/live-response?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/live-response.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/live-response.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @diannegali
* Microsoft Alias: **diannegali**

### PR DESCRIPTION
**Pull request by msbemba**
Changed description for scan

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/13206

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/live-response.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ The following commands are available for user roles that are granted the ability
 | library  | Lists files that were uploaded to the live response library.  | Y  | Y  | Y  |
 | putfile  | Puts a file from the library to the device. Files are saved in a working folder and are deleted when the device restarts by default.  | Y  | Y  | Y  |
 | remediate  | Remediates an entity on the device. The remediation action will vary depending on the entity type:  File: delete  Process: stop, delete image file  Service: stop, delete image file  Registry entry: delete  Scheduled task: remove  Startup folder item: delete file  NOTE: This command has a prerequisite command. You can use the -auto command in conjunction with remediate to automatically run the prerequisite command.  | Y  | Y  | Y  |
-| scan | Runs an antivirus scan to help identify and remediate malware. | N | Y | Y |
+| scan | Runs an Quick antivirus scan to help identify and remediate malware. | N | Y | Y |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/live-response.md:187]:
```suggestion
| scan | Runs a Quick antivirus scan to help identify and remediate malware. | N | Y | Y |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@diannegali Please merge this PR. Thanks

**Pull request merged**